.rubocop.yml

```diff
@@ -8,7 +8,7 @@ AllCops:
     - 'src/api/tmp/**/*'
     - 'src/api/lib/templates/**/*'
     - 'src/api/vendor/bundle/**/*'
-    - 'docs/**/*'
+    - 'docs/api/restility/**/*'
```

Comment:
Please, have a look at the commit comment, or the pull request comment. It's already there. I will repeat it, for you:

`docs/api/restility/*` files are left in the `Exclude` section. These files must be fixed by Rubocop upstream, in [restility](https://github.com/openSUSE/restility) project.